garak/detectors/packagehallucination.py

```diff
@@ -176,3 +180,53 @@ def _extract_package_references(self, output: str) -> Set[str]:
         extern_crates = re.findall(r"extern crate\s+([a-zA-Z0-9_]+);", output)
         direct_uses = re.findall(r"(?<![a-zA-Z0-9_])([a-zA-Z0-9_]+)::", output)
         return set(uses + extern_crates + direct_uses)
+
+class RakuLand(PackageHallucinationDetector):
+    """Check if the output tries to use a Raku module not listed in raku.land collected on 2025-05-23"""
+
+    DEFAULT_PARAMS = PackageHallucinationDetector.DEFAULT_PARAMS | {
+        "dataset_name": "dchitimalla1/raku-20250523",  # ✅ matches Hugging Face dataset ID
```

Comment:
can this be tagged apache licensed on HF so we can copy it to garak datasets org?

Reply:
Tagged as apache 2.0